app/styles/modules/_settings-totp.scss

```diff
@@ -138,6 +138,15 @@
   margin-top: 25px;
 }
 
+.save-option-button {
+  background: none !important;
```

Comment:
Try to not use `!important` unless absolutely necessary, it makes the assumption that we'll *never* want to override this style. It's far preferable is to use a more specific selector. See https://css-tricks.com/when-using-important-is-the-right-choice/ for reference.

Reply:
Following on, we'll want the :focus state of these buttons to mirror the :focus state on the existing links and look like:

<img width="166" alt="Screenshot 2019-03-26 at 10 36 53" src="https://user-images.githubusercontent.com/848085/54990673-334dfc80-4fb3-11e9-85d6-e123217c6779.png">

Currently the :focus looks like:

<img width="158" alt="Screenshot 2019-03-26 at 10 35 15" src="https://user-images.githubusercontent.com/848085/54990733-52e52500-4fb3-11e9-887c-543e958ee187.png">